applications/StructuralMechanicsApplication/custom_conditions/small_displacement_line_load_condition.h

```diff
@@ -6,20 +6,19 @@
 //  License:		 BSD License
 //					 license: structural_mechanics_application/license.txt
 //
-//  Main authors:    Riccardo Rossi
-//                   Vicente Mataix Ferrandiz
+//  Main authors:    Vicente Mataix Ferrandiz
```

Comment:
what about the "large_deformation" versions? line_load was working correctly in large deformations...

Reply:
Yes, it is

Reply:
@RiccardoRossi GitHub detects this as a name  change, but line_load_condition_2d.h  is removed as it is deprecated (and not actually used since long time ago)

Reply:
@rubenzorrilla  see like Github detects a name change (->), and see the previous conversation:

![imagen](https://user-images.githubusercontent.com/19991680/73792016-c0510d00-47a3-11ea-9484-e2b661e0e966.png)

Reply:
OK. Fine then.

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/35867376-e2292dfa-0b59-11e8-8ec9-db7ad94a9d5c.png)